### ISSUE TITLE
Target classes inherited last

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#601]
As upstream warns about if any of the following

- native
- nativesdk
- cross

are not inherited last, we should issues a warning too.

E.g.

´´´
inherit native
inherit foo
```

should raise a warning about unknown side effects of the operation
